imagetest/test_suites/network/gvnic_test.go

```diff
@@ -45,8 +50,27 @@ func CheckGVNICPresentWindows(interfaceName string) error {
 	return errors.New("GVNIC not present")
 }
 
+func CheckGVNICPerformance() (string, error) {
+	// Wait for iperf to finish.
+	for i := 0; i <= maxRetries; i++ {
```

Comment:
If I understand this code correctly you are waiting until the `iperf` finishes, right? In that case is the `10 * (60 * time.Second)` duration aligned with the iperf test timeout?

Reply:
Yea. As for the second question, I will discuss with you privately. Basically, the same interval was used in a similar test somewhere else, so I've decided to reuse the same interval.

Reply:
Should be fine. The retry logic looks correct, just wonder if the times are realistic - but they can be throttled later with more "real data".